pipeline/test_load.py

```diff
@@ -17,16 +16,51 @@ def test_insert_price_change_valid():
         22.49, 1, "2024-12-04 16:31:40")
 
 
+# def test_load_price_changes_valid():
```

Comment:
Does this test need to be commented out? If yes, then it should be deleted. If no, then it should be included.